sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -87,10 +87,9 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
       case "overwrite" => mode(SaveMode.Overwrite)
       case "append" => mode(SaveMode.Append)
       case "ignore" => mode(SaveMode.Ignore)
-      case "error" | "errorifexists" => mode(SaveMode.ErrorIfExists)
-      case "default" => this
```

Comment:
this looks suspicious. looks like we never document "default" option.

Reply:
Yes. It seems that we didn't document it before.